jibri/rootfs/defaults/jibri.conf

```diff
@@ -0,0 +1,58 @@
+jibri {
+    api {
+        http {
+            external-api-port = {{ .Env.JIBRI_HTTP_API_EXTERNAL_PORT | default 2222 }}
+            internal-api-port = {{ .Env.JIBRI_HTTP_API_INTERNAL_PORT | default 3333 }}
+        }
+        xmpp {
+            environments = [
+   		        {
+                    name = "prod environment"
+                    xmpp-server-hosts = [
+                        "{{ .Env.XMPP_SERVER }}"
```

Comment:
Allow to add multiple XMPP servers. We can configure `XMPP_SERVER =xmpp1,xmpp2` using env variables. You can use something like this `"{{ join "\",\n\"" (splitList "," .Env.XMPP_SERVER }}"`